Pyrado/pyrado/sampling/parameter_exploration_sampler.py

```diff
@@ -126,9 +127,9 @@ def _pes_init(G, env, policy):
     G.policy = pickle.loads(policy)
 
 
-def _pes_sample_one(G, param):
+def _pes_sample_one(G, param, seed: str):
     """Sample one rollout with the current setting."""
-    pol_param, dom_param, init_state = param
+    num, (pol_param, dom_param, init_state) = param
```

Comment:
wouldn't it be better to pass `num` as an arg here?

Reply:
I'll take a closer look at the `run_map` method of this would be easier. Given my comment to your other annotation,im I have to touch this again nevertheless.

Reply:
I checked again, but `run_map` only supports one argument to be passed to the subroutine.